src/main/java/org/jivesoftware/smackx/pubsub/PubSubIntegrationTest.java

```diff
@@ -947,4 +947,49 @@ public void deleteNonExistentNodeTest() throws NoResponseException, XMPPErrorExc
             assertEquals(StanzaError.Condition.item_not_found, e.getStanzaError().getCondition());
         }
     }
+
+    /**
+     * Assert that the server send a notification to subscribers when deleting a
+     * node that exist
+     * 
+     * <p>
+     * From XEP-0060 § 8.4.2:
+     * </p>
+     * <blockquote> In order to delete a node, a node owner MUST send a node
+     * deletion request, consisting of a &lt;delete/&gt; element whose 'node'
+     * attribute specifies the NodeID of the node to be deleted </bloquote>
+     * 
+     * @throws NoResponseException                     if there was no response from
+     *                                                 the remote entity.
+     * @throws XMPPErrorException                      if there was an XMPP error
+     *                                                 returned.
+     * @throws NotConnectedException                   if the XMPP connection is not
+     *                                                 connected.
+     * @throws InterruptedException                    if the calling thread was
+     *                                                 interrupted.
+     * @throws PubSubException.NotAPubSubNodeException if the node cannot be
+     *                                                 accessed.
+     */
+    @SmackIntegrationTest
+    public void deleteNodeAndNotifySubscribersTest() throws NoResponseException, XMPPErrorException,
+        NotConnectedException, InterruptedException, TimeoutException, ExecutionException {
+        final String nodename = "sinttest-delete-node-that-exist-" + testRunId;
+        final String needle = "<event xmlns='http://jabber.org/protocol/pubsub#event'>";
+        final String delete_confirm = "<delete node='princely_musings'>";
```

Comment:
This won't work. You're creating a node wit a name that starts with `sinttest-delete-node-that-exists-`, but you're deleting a node that is named `princely_musings`.